### ISSUE TITLE
t/basic.t and t/test-compile.t fail (with Type::Tiny 2.000000?)

### ISSUE THREAD

**Issue by eserte** [Ovid/moosex-extended#58]
The test suite of MooseX-Extended-0.32 fails on most of my smokers:
```
t/basic.t ................. 
Dubious, test returned 255 (wstat 65280, 0xff00)
No subtests run 
...
# Could not find sub 'signature' exported by MooseX::Extended::Types at /usr/perl5.22.1p/lib/site_perl/5.22.1/Import/Into.pm line 40.
# BEGIN failed--compilation aborted at t/basic.t line 7.

#   Failed test 't/basic.t compiles'
#   at t/test-compile.t line 50.
# Looks like you failed 1 test of 35.
t/test-compile.t .......... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/35 subtests 
...
```

Statistical analysis suggests that this happens if Type::Tiny 2.000000 is installed.

**Comment by Ovid** [comment]
I've been seeing that. I have _no_ idea why this is happening. [I have a PR that seems to fix the issue](https://github.com/Ovid/moosex-extended/pull/57), but again, no idea what's on on. In fact, I did `git grep '\<signature\>'` and the only instance of that was in a comment! So this should be fixed, but I'm mystified as to why it's even happening.

**Comment by Ovid** [comment]
As an aside, I could never replicate the failure locally. However, this is fixed on version 0.33 on its way to the CPAN now.
